src/Microsoft.AspNetCore.SignalR/HubEndPoint.cs

```diff
@@ -120,6 +120,14 @@ public class HubEndPoint<THub> : IInvocationBinder where THub : Hub
                             ? (IDataEncoder)Base64Encoder
                             : PassThroughEncoder;
 
+                        var transferMode = connection.Features.Get<ITransferModeFeature>() ??
```

Comment:
I know understand how this sneaked in - https://github.com/aspnet/SignalR/pull/711